bblfsh/aliases.py

```diff
@@ -1,27 +1,36 @@
-__all__ = ["DESCRIPTOR", "Node", "Position", "ParseResponse", "NativeParseResponse",
-           "ParseRequest", "NativeParseRequest", "VersionRequest", "ProtocolServiceStub"]
-
 import importlib
-
-from bblfsh.sdkversion import VERSION
+import google
 
 # "in" is a reserved keyword in Python thus can't be used as package name, so
 # we import by string
-uast_module = importlib.import_module(
-    "bblfsh.gopkg.in.bblfsh.sdk.%s.uast.generated_pb2" % VERSION)
-protocol_module = importlib.import_module(
-    "bblfsh.gopkg.in.bblfsh.sdk.%s.protocol.generated_pb2" % VERSION)
-protocol_grpc_module = importlib.import_module(
-    "bblfsh.gopkg.in.bblfsh.sdk.%s.protocol.generated_pb2_grpc" % VERSION)
+uast_v2_module = importlib.import_module(
+    "bblfsh.gopkg.in.bblfsh.sdk.v2.uast.generated_pb2")
+protocol_v2_module = importlib.import_module(
+    "bblfsh.gopkg.in.bblfsh.sdk.v2.protocol.generated_pb2")
+protocol_grpc_v2_module = importlib.import_module(
+    "bblfsh.gopkg.in.bblfsh.sdk.v2.protocol.generated_pb2_grpc")
+protocol_v1_module = importlib.import_module(
+    "bblfsh.gopkg.in.bblfsh.sdk.v1.protocol.generated_pb2")
+protocol_grpc_v1_module = importlib.import_module(
+    "bblfsh.gopkg.in.bblfsh.sdk.v1.protocol.generated_pb2_grpc")
+
+DESCRIPTOR = uast_v2_module.DESCRIPTOR
+ParseRequest = protocol_v2_module.ParseRequest
+ParseResponse = protocol_v2_module.ParseResponse
+ParseError = protocol_v2_module.ParseError
+Mode = protocol_v2_module.Mode
+ModeType = google.protobuf.internal.enum_type_wrapper.EnumTypeWrapper
+
+# Current values: {'DEFAULT_MODE': 0, 'NATIVE': 1, 'PREPROCESSED': 2, 'ANNOTATED': 4, 'SEMANTIC': 8}
+ModeDict = {}
```

Comment:
```suggestion
class Modes:
    pass
for k, v in Mode.DESCRIPTOR.values_by_name.items():
    setattr(Modes, k, v)
```

Reply:
notbad.jpg :+1:

Reply:
(will commit separately since github doesn't allow me to commit suggestions to this branch)